### PR TITLE
docs: fix forward-auth plugin doc

### PR DESCRIPTION
**Pull request by Gallardot**
### What this PR does / why we need it:

Fixed some curl usage errors.

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by spacewander — DISMISSED** [review]

File: docs/en/latest/plugins/forward-auth.md [hunk 1]
```diff
@@ -117,32 +117,35 @@ We can perform the following three tests.
 1. **request_headers** Send Authorization header from `client` to `authorization` service
 
 ```shell
-$ curl http://127.0.0.1:9080/headers -H 'Authorization: 123'
-{
+curl http://127.0.0.1:9080/headers -H 'Authorization: 123' \
+-d '{
```
Comment (bzp2010) [inline docs/en/latest/plugins/forward-auth.md:121]: Incorrent modification, this is the response.

File: docs/en/latest/plugins/forward-auth.md [hunk 2]
```diff
@@ -117,32 +117,35 @@ We can perform the following three tests.
...
+}'
 ```
 
 2. **upstream_headers** Send `authorization` service response header to the `upstream`
 
 ```shell
-$ curl http://127.0.0.1:9080/headers -H 'Authorization: 321'
-{
+curl http://127.0.0.1:9080/headers -H 'Authorization: 321' \
+-d '{
```
Comment (bzp2010) [inline docs/en/latest/plugins/forward-auth.md:133]: ditto

**Review by bzp2010 — CHANGES_REQUESTED** [review]

File: docs/en/latest/plugins/forward-auth.md (hunk 1 above)
Comment (Gallardot) [reply docs/en/latest/plugins/forward-auth.md]: Good catch! I changed this to two text blocks. PTAL!

File: docs/en/latest/plugins/forward-auth.md (hunk 2 above)
Comment (Gallardot) [reply docs/en/latest/plugins/forward-auth.md]: good catch! I changed this to two text blocks. PTAL!

**Review by spacewander — APPROVED** [review]

**Review by starsz — APPROVED** [review]

**Review by oil-oil — APPROVED** [review]

**Review by leslie-tsang — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Pull request merged**